app/controllers/application_controller.rb

```diff
@@ -50,8 +50,8 @@ def authenticate
 
   def authenticate_token
     authenticate_with_http_token do |token, _options|
-      if (@current_user = User.find_by(api_key: token))
-        @current_user.identity = UserAttributes.new(username: @current_user.username)
+      if (@current_user = User.find_by(api_key_digest: Keycard::ApiKey.new(token).digest))
+        @current_user.identity = UserAttributes.new(username: @current_user.username).all
```

Comment:
I want to check with @aelkiss on this, but I'm not sure we need the JWT support at all. We could definitely support JWT, but it seems to me that the built-in support is much more in line with browser workflows with third party / social sign-in, neither of which seems relevant here.

Reply:
No, there's no JWT support required, but I'm not sure exactly what this comment is referring to in the context of the code.

Reply:
Per our discussion this morning, merging as-is will cause no trouble, but the Rails-provided authenticate_with_http_token is not really needed. We should trim this down and simplify.